samples/zigbee/ncp/prj.conf

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+# Enable nRF ECB driver
+CONFIG_CRYPTO=y
+CONFIG_CRYPTO_NRF_ECB=y
+
+# Cryptocell is not supported through CSPRNG driver API: NCSDK-4813
+CONFIG_ENTROPY_CC3XX=n
+
+CONFIG_ZIGBEE=y
+CONFIG_ZIGBEE_LIBRARY_DEVELOPMENT=y
+
+# Select which featuresets should be built
```

Comment:
Fix the comment. The Kconfig below does not select featureset to be built - it includes the NCP -DEV library to the app instead.